src/concepts/pipeline/event_pipeline.md

```diff
@@ -90,16 +91,15 @@ use cases:
 - The [*dynamic event API*](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/events.html#registerevents)
   allows code from Mozilla addons to record new events into Telemetry without shipping Firefox
   code.
-- The *Telemetry extension API* ([work in progress](https://bugzilla.mozilla.org/show_bug.cgi?id=1280234))
-  will allow Mozilla extensions to record new events into Telemetry.
+- The *[Telemetry WebExtension API](https://searchfox.org/mozilla-central/rev/55da592d85c2baf8d8818010c41d9738c97013d2/toolkit/components/extensions/schemas/telemetry.json#87)* ([introduced here](https://bugzilla.mozilla.org/show_bug.cgi?id=1280234))
```

Comment:
There's no such thing as WebExtensions. Only "Extensions" and "Legacy Extensions"

Reply:
It's a confusing situation. We talk about "extensions"/"browser extensions" internally, but still call the API [the "WebExtension API" ](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions).
I think i'll just leave this as is for now, it should be clear enough i hope.

Reply:
The title of that page is "Browser Extensions" :) 

It was a deliberate move around 57 or so when we were removing Legacy Extensions to specifically call the newer ones just Extensions. I would like to press this matter.

Reply:
The page talks about the WebExtensions API though.

Reply:
Also [here](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API).

Reply:
Clarified that WebExtension API is a term still.
Clarified that there is a separate concern about not calling the extensions themselves WebExtensions.